crates/bevy_gltf/src/loader.rs

```diff
@@ -978,4 +983,50 @@ mod test {
         assert_eq!(result[0].0, "l2");
         assert_eq!(result[0].1.children.len(), 0);
     }
+
+    use super::Gltf;
+    use bevy_app::{App, AppExit, EventWriter};
+    use bevy_asset::{AssetPlugin, AssetServer, Assets, Handle};
+    use bevy_core::CorePlugin;
+    use bevy_ecs::system::{Commands, Res};
+    use bevy_scene::Scene;
+
+    #[test]
+    fn test_scene_to_nodes() {
+        App::new()
+            .add_plugin(bevy_log::LogPlugin::default())
+            .add_plugin(CorePlugin::default())
+            .add_plugin(AssetPlugin::default())
+            .add_plugin(bevy_scene::ScenePlugin::default())
+            .add_plugin(GltfPlugin::default())
```

Comment:
We don't have access to `DefaultPlugins` here, or at least I couldn't figure out a way to do this.
Should we add a test only dependency?

Reply:
I added it as a dev dependency